config/routes.rb

```diff
@@ -3,7 +3,7 @@
   mount Blacklight::Oembed::Engine, at: 'oembed'
   mount Riiif::Engine => '/images', as: 'riiif'
   root to: 'spotlight/exhibits#index'
-  mount Spotlight::Engine, at: 'spotlight'
+  mount Spotlight::Engine, at: '/'
```

Comment:
@starsplatter No ill effects from mounting both Blacklight and Spotlight at `/`?